### PR TITLE
fix: remove overlapping of text and icons

### LINKED ISSUE #9461: Overlapping in text and icons
**Describe the bug**
Icons and text in the shifting page are overlapping and looking messy.

**To Reproduce**

1. Go to ['https://care.ohc.network/shifting/list'](url)
2. See in status column.

**Expected behavior**
Text should not be overlap with the icons. We can improve it using text wrap for status column. Also we have to move tags like 'emergency' to below the current status.

**Screenshots**

Image

**Desktop (please complete the following information):**

- OS: MacOS
- Browser Chrome

### PR DESCRIPTION
**Pull request by Tanuj1718**
## Proposed Changes

- Fixes #9461
- Remove overlapping of text and icons

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [x] Completion of QA

## Summary by CodeRabbit

- **Style**
	- Updated class names in the ShiftingTable component to enhance layout and responsiveness.
- **Bug Fixes**
	- Maintained existing error handling and control flow for transfer completion.

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
image

**Comment by Tanuj1718** [comment]
> image

I am not able to see this overflow of text in my device. Can you please tell more?

Screenshot 2024-12-18 at 11 28 47 PM

**Comment by Jacobjeevan** [comment]
> > image
> 
> I am not able to see this overflow of text in my device. Can you please tell more?
> Screenshot 2024-12-18 at 11 28 47 PM

Check using your browser's device mode and try different viewports (for example: iPad layout horizontal and you will see this).

**Review by Jacobjeevan — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

File: src/components/Shifting/ShiftingTable.tsx [hunk 1]
```diff
@@ -134,8 +134,8 @@ export default function ShiftingTable(props: {
                 </div>
               </div>
 
-              <div className="col-span-1 flex flex-col px-3 text-left">
-                <div className="3xl:flex-row mb-2 flex gap-2 sm:flex-row md:flex-row lg:flex-col xl:flex-row 2xl:flex-row">
+              <div className="col-span-1  flex flex-col text-left">
```
Comment (rithviknishad) [inline src/components/Shifting/ShiftingTable.tsx]:
```suggestion
              <div className="col-span-1 flex flex-col text-left">
```

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**